### PR TITLE
Automated cherry pick of #1795: Remove <v1.25 pod monitor

### PR DESCRIPTION
**Pull request by rene-dekker**
Cherry pick of #1795 on release-v1.25.

#1795: Remove <v1.25 pod monitor

### CODE REVIEW

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Comment by marvin-tigera** [comment]
Removing "merge-when-ready" label due to new commits

**Review by tmjd — APPROVED** [review]
LGTM

**Pull request merged**
